connect/transforms/src/test/java/org/apache/kafka/connect/transforms/DropHeadersTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.connect.transforms;
+
+import org.apache.kafka.connect.header.ConnectHeaders;
+import org.apache.kafka.connect.header.Headers;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class DropHeadersTest {
+    private DropHeaders<SourceRecord> xform = new DropHeaders<>();
+
+    @After
+    public void teardown() {
+        xform.close();
+    }
```

Comment:
How about adding some tests that check that calling `DropHeaders.configure(props)` with invalid properties results in a `ConfigException`? For example:
```
@Test(expected = ConfigException.class)
public void shouldFailWithBlankHeaderNames() {
    Map<String, String> props = new HashMap<>();
    props.put("names", "");
    transform.config(props);
}

@Test(expected = ConfigException.class)
public void shouldFailWithBlankHeaderNames() {
    Map<String, String> props = new HashMap<>();
    props.put("names", "a, ");  // <-- This should fail, but it succeeds now
    transform.config(props);
}

@Test(expected = ConfigException.class)
public void shouldFailWithBlankHeaderNames() {
    Map<String, String> props = new HashMap<>();
    props.put("names", "a,,d");  // <-- This should fail, but it succeeds now
    transform.config(props);
}```